src/sinks/papertrail.rs

```diff
@@ -81,6 +81,10 @@ impl SinkConfig for PapertrailConfig {
     fn sink_type(&self) -> &'static str {
         "papertrail"
     }
+
+    fn can_acknowledge(&self) -> bool {
+        false
```

Comment:
Same question, can Papertrail accept data, but drop it?

Reply:
I don't see that we're handling any acknowledgements from Papertrail in our sink.

Reply:
Ah, yes, I forgot the Papertrail integration was just using raw TCP.